hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceWALReader.java

```diff
@@ -267,10 +267,11 @@ public Path getCurrentPath() {
   //returns false if we've already exceeded the global quota
   private boolean checkQuota() {
     // try not to go over total quota
-    if (source.manager.getTotalBufferUsed().get() > source.manager.getTotalBufferLimit()) {
+    if (source.overallController.getTotalBufferUsed().get() > source.overallController
```

Comment:
Yeah, if the Controller is in the ReplicationSourceManager, then the 'overall' will be redundant?